src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/activities/ActivityToNomisIntTest.kt

```diff
@@ -57,7 +57,7 @@ class ActivityToNomisIntTest : SqsIntegrationTestBase() {
       activitiesApi.stubGetActivity(ACTIVITY_ID, buildGetActivityResponse())
       mappingServer.stubGetMappingGivenActivityScheduleIdWithError(ACTIVITY_SCHEDULE_ID, 404)
       mappingServer.stubCreateActivity()
-      nomisApi.stubActivityCreate("""{ "courseActivityId": $NOMIS_CRS_ACTY_ID }""")
+      nomisApi.stubActivityCreate("""{ "courseActivityId": $NOMIS_CRS_ACTY_ID, "courseSchedules": [] }""")
```

Comment:
We're currently ignoting the response contents, this PR is just to update the response model. Next PR will do something with the response.